app/res/layout/fragment_setup_enter_url.xml

```diff
@@ -69,14 +69,12 @@
                     android:layout_width="fill_parent"
                     android:layout_height="wrap_content"
                     android:paddingTop="@dimen/content_start"
+                    android:enabled="false"
                     android:paddingBottom="@dimen/content_start"
                     android:layout_marginTop="@dimen/content_min_margin"
                     android:layout_gravity="bottom"
-                    android:textColor="@color/cc_neutral_bg"
-                    android:background="@color/cc_brand_color"
                     android:text="Start Install"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/5a959df2d1681e0a640ebb72d4513eaecefa585e/app/res/layout/fragment_setup_enter_url.xml#L76"></span>Hardcoded string "Start Install", should use <code>@string</code> resource</td>
    </tr>
  </tbody>
</table>